### PR TITLE
Made the name of environment variables consistent through benchmarking doc

### PR DESCRIPTION
**Pull request by sauraank**
## Description of change
In our [Benchmarking doc](https://github.com/awslabs/mountpoint-s3/blob/main/doc/BENCHMARKING.md#running-the-benchmark), the names for environment variables `S3_BUCKET_NAME`, `S3_BUCKET_TEST_PREFIX`, `S3_BUCKET_BENCH_FILE` and `S3_BUCKET_SMALL_BENCH_FILE` across steps. This might create confusion for new user trying to run benchmark.

Made the names consistent across the steps for environment variables in used in benchmarking.

Relevant issues: 
NA

## Does this change impact existing behavior?
No. Just documentation edits.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and I agree to the terms of the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).

### CODE REVIEW

File: doc/BENCHMARKING.md [hunk 1]
```diff
@@ -32,12 +32,12 @@ You can use the following steps.
                 --fuse-version 2 \
                 --with-fio --with-libunwind
 
-2. Set environment variables related to the benchmark. There are four required environment variables you need to set in order to run the benchmark.
+2. Set environment variables related to the benchmark. There are four required environment variables you need to set in order to run the benchmark. These variables can be set to any name, but be consistent with it in subsequent steps.
```
Comment (arsh) [inline doc/BENCHMARKING.md:35]: Should we change these steps to first create the files needed and then populate the env variables?

Also, on line 45, there is a `DOC-EXAMPLE-BUCKET` which is undefined. Please add that to the set of variables to set.

Comment (sauraank) [reply doc/BENCHMARKING.md:35]: Yes. That seems more reasonable. Making the change. Thanks

File: doc/BENCHMARKING.md [hunk 2]
```diff
@@ -52,6 +45,13 @@ You can use the following steps.
         fio --directory=$MOUNT_DIR/bench_dir_10000 create/create_files_10000.fio
         fio --directory=$MOUNT_DIR/bench_dir_100000 create/create_files_100000.fio
 
+3. Set environment variables related to the benchmark. There are four required environment variables you need to set in order to run the benchmark. These variable names should be the same as the resource names used in the previous step.
+
+        export S3_BUCKET_NAME=DOC-EXAMPLE-BUCKET
+        export S3_BUCKET_TEST_PREFIX=benchmark/
+        export S3_BUCKET_BENCH_FILE=bench100GB.bin
+        export S3_BUCKET_SMALL_BENCH_FILE=bench5MB.bin
```
[the remark this answers is not in the document]
Comment (sauraank) [reply doc/BENCHMARKING.md:53]: These environment are to run the benchmark script that we run in step 4. We earlier thought that these environment variables should be set once we have created the resource: https://github.com/awslabs/mountpoint-s3/pull/740#discussion_r1482761964

**Comment by dannycjones** [comment]
This change will be superseded by #1108.

**Pull request closed without merging**